schemas/index.js

```diff
@@ -1 +1,13 @@
-export const schemas = new Map([]);
+import * as defaultTokenAddress from "./defaultTokenAddress.js";
+
+export const schemas = new Map([
+  [
+    "defaultTokenAddress",
```

Comment:
For DSNP, we've been using kebab-style naming (e.g. `dsnp.profile-resources`); if `frequency.*` wants to do it differently I don't think it really matters, but thought I'd bring it up. I also wonder if, because the itemized structure can support multiple addresses, it should be "addresses" rather "address" for the schema name (we moved that direction for DSNP to make the usage easier to understand for things like public keys). Although it does then make it strange to call it "default" if there's (potentially) more than one? Is there something like "knownTokenAddresses" or "preferredTokenAddresses" that could work better semantically?

Reply:
I'm good with the kebab style. There was a lot of discussion about the naming. This is the one that H.E. thought was best of the suggestions. Open to new ones if you can convince H.E.
```suggestion
    "default-token-address",
```